aws/ami/windows/scripts/Installers/Install-CUDA-Tools.ps1

```diff
@@ -83,12 +83,20 @@ function Install-Cudnn() {
   Invoke-WebRequest -Uri "$windowsS3BaseUrl/$cudnnZip" -OutFile "$tmpCudnnInstall"
   $tmpCudnnExtracted = New-TemporaryDirectory
   7z x "$tmpCudnnInstall" -o"$tmpCudnnExtracted"
-  Write-Output "Copying cudnn to $expectedInstallLocation"
-  Copy-Item -Force -Verbose -Recurse "$tmpCudnnExtracted\cuda\*" "$expectedInstallLocation"
-  if (-Not (Test-Path -Path "$expectedInstallLocation\include\cudnn.h" -PathType Leaf)) {
-    Write-Error "cudnn installation failed for CUDA version $cudaVersion, cudnn.h not found at $expectedInstallLocation\include\cudnn.h"
-    exit 1
+
+
+  if($cudaVersion == "11.5") {
```

Comment:
Is there any way to do this without the special casing just for CUDA 11.5?

If we can have generic logic for everything it'll be easier to maintain

Reply:
Done!